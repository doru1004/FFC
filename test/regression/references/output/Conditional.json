{
    "form_0": {
        "num_coefficients": 1, 
        "has_cell_integrals": 1, 
        "has_exterior_facet_integrals": 0, 
        "has_interior_facet_integrals": 0, 
        "has_point_integrals": 0, 
        "num_cell_domains": 0, 
        "num_exterior_facet_domains": 0, 
        "num_interior_facet_domains": 0, 
        "num_point_domains": 0, 
        "finite_element_0": {
            "cell_shape": 1, 
            "space_dimension": 6, 
            "value_rank": 0, 
            "evaluate_basis:_0": [2.861007167858017], 
            "evaluate_basis:_1": [6.188116389316917], 
            "evaluate_basis:_2": [14.98411813143485], 
            "evaluate_basis:_3": [-20.25499781634912], 
            "evaluate_basis:_4": [-14.71034988724643], 
            "evaluate_basis:_5": [11.93210574240425], 
            "evaluate_basis_all": [2.861007167858017, 6.188116389316917, 14.98411813143485, -20.25499781634912, -14.71034988724643, 11.93210574240425], 
            "evaluate_basis_derivatives_0_0": [2.861007167858017], 
            "evaluate_basis_derivatives_0_1": [0.4183016632598591, -11.13728113944124], 
            "evaluate_basis_derivatives_0_2": [0.02929918488607441, -0.7800958379073109, -0.7800955273458143, 20.770049714109], 
            "evaluate_basis_derivatives_1_0": [6.188116389316917], 
            "evaluate_basis_derivatives_1_1": [-16.80228443241919, 3.193194233653243], 
            "evaluate_basis_derivatives_1_2": [22.35954107583998, -4.24932433998028, -4.249324213791152, 0.8075636705377246], 
            "evaluate_basis_derivatives_2_0": [14.98411813143485], 
            "evaluate_basis_derivatives_2_1": [-25.05263017019036, -20.11267492536409], 
            "evaluate_basis_derivatives_2_2": [20.7700505623934, 16.67454763459752, 16.67454763459752, 13.38660866439519], 
            "evaluate_basis_derivatives_3_0": [-20.25499781634912], 
            "evaluate_basis_derivatives_3_1": [42.10204576699874, 10.33961641517088], 
            "evaluate_basis_derivatives_3_2": [-43.10029269019528, -13.20531909572782, -13.20531891135545, 6.575877590910224], 
            "evaluate_basis_derivatives_4_0": [-14.71034988724643], 
            "evaluate_basis_derivatives_4_1": [12.56063157739744, 33.54450483314719], 
            "evaluate_basis_derivatives_4_2": [1.560191565408481, -20.14377617346721, -20.14377635783958, -33.3490949197006], 
            "evaluate_basis_derivatives_5_0": [11.93210574240425], 
            "evaluate_basis_derivatives_5_1": [-13.22606377112947, -15.82735912223852], 
            "evaluate_basis_derivatives_5_2": [-1.618790685187193, 21.70396796518927, 21.70396752843863, -8.191005649914606], 
            "evaluate_basis_derivatives_all_0": [2.861007167858017, 6.188116389316917, 14.98411813143485, -20.25499781634912, -14.71034988724643, 11.93210574240425], 
            "evaluate_basis_derivatives_all_1": [0.4183016632598591, -11.13728113944124, -16.80228443241919, 3.193194233653243, -25.05263017019036, -20.11267492536409, 42.10204576699874, 10.33961641517088, 12.56063157739744, 33.54450483314719, -13.22606377112947, -15.82735912223852], 
            "evaluate_basis_derivatives_all_2": [0.02929918488607441, -0.7800958379073109, -0.7800955273458143, 20.770049714109, 22.35954107583998, -4.24932433998028, -4.249324213791152, 0.8075636705377246, 20.7700505623934, 16.67454763459752, 16.67454763459752, 13.38660866439519, -43.10029269019528, -13.20531909572782, -13.20531891135545, 6.575877590910224, 1.560191565408481, -20.14377617346721, -20.14377635783958, -33.3490949197006, -1.618790685187193, 21.70396796518927, 21.70396752843863, -8.191005649914606], 
            "evaluate_dof_0": 0.3079230000000001, 
            "evaluate_dof_1": 0.430287, 
            "evaluate_dof_2": 0.772821, 
            "evaluate_dof_3": 0.59985, 
            "evaluate_dof_4": 0.5310900000000001, 
            "evaluate_dof_5": 0.405528, 
            "evaluate_dofs": [0.3079230000000001, 0.430287, 0.772821, 0.59985, 0.5310900000000001, 0.405528], 
            "interpolate_vertex_values": [0.3079230000000001, 0.430287, 0.772821], 
            "num_sub_elements": 0, 
        },
        "finite_element_1": {
            "cell_shape": 1, 
            "space_dimension": 1, 
            "value_rank": 0, 
            "evaluate_basis:_0": [1], 
            "evaluate_basis_all": [1], 
            "evaluate_basis_derivatives_0_0": [1], 
            "evaluate_basis_derivatives_0_1": [0.0, 0.0], 
            "evaluate_basis_derivatives_0_2": [0.0, 0.0, 0.0, 0.0], 
            "evaluate_basis_derivatives_all_0": [1], 
            "evaluate_basis_derivatives_all_1": [0.0, 0.0], 
            "evaluate_basis_derivatives_all_2": [0.0, 0.0, 0.0, 0.0], 
            "evaluate_dof_0": 0.5149823230336866, 
            "evaluate_dofs": [0.5149823230336866], 
            "interpolate_vertex_values": [0.5149823230336866, 0.5149823230336866, 0.5149823230336866], 
            "num_sub_elements": 0, 
        },
        "dofmap_0": {
            "needs_mesh_entities_0": 1, 
            "needs_mesh_entities_1": 1, 
            "needs_mesh_entities_2": 0, 
            "global_dimension": 20003, 
            "local_dimension": 6, 
            "max_local_dimension": 6, 
            "geometric_dimension": 2, 
            "num_facet_dofs": 3, 
            "num_entity_dofs_0": 1, 
            "num_entity_dofs_1": 1, 
            "num_entity_dofs_2": 0, 
            "tabulate_dofs": [0, 0, 0, 10001, 10002, 10003], 
            "tabulate_facet_dofs_0": [1, 2, 3], 
            "tabulate_facet_dofs_1": [0, 2, 4], 
            "tabulate_facet_dofs_2": [0, 1, 5], 
            "tabulate_entity_dofs_0_0": [0], 
            "tabulate_entity_dofs_0_1": [1], 
            "tabulate_entity_dofs_0_2": [2], 
            "tabulate_entity_dofs_1_0": [3], 
            "tabulate_entity_dofs_1_1": [4], 
            "tabulate_entity_dofs_1_2": [5], 
            "tabulate_entity_dofs_2_0": [], 
            "tabulate_coordinates_0": [0.903, 0.341], 
            "tabulate_coordinates_1": [0.5610000000000001, 0.767], 
            "tabulate_coordinates_2": [0.987, 0.783], 
            "tabulate_coordinates_3": [0.774, 0.775], 
            "tabulate_coordinates_4": [0.9450000000000001, 0.5620000000000001], 
            "tabulate_coordinates_5": [0.732, 0.554], 
            "num_sub_dofmaps": 0, 
        },
        "dofmap_1": {
            "needs_mesh_entities_0": 0, 
            "needs_mesh_entities_1": 0, 
            "needs_mesh_entities_2": 0, 
            "global_dimension": 1, 
            "local_dimension": 1, 
            "max_local_dimension": 1, 
            "geometric_dimension": 2, 
            "num_facet_dofs": 0, 
            "num_entity_dofs_0": 0, 
            "num_entity_dofs_1": 0, 
            "num_entity_dofs_2": 1, 
            "tabulate_dofs": [0], 
            "tabulate_facet_dofs_0": [], 
            "tabulate_facet_dofs_1": [], 
            "tabulate_facet_dofs_2": [], 
            "tabulate_entity_dofs_0_0": [], 
            "tabulate_entity_dofs_0_1": [], 
            "tabulate_entity_dofs_0_2": [], 
            "tabulate_entity_dofs_1_0": [], 
            "tabulate_entity_dofs_1_1": [], 
            "tabulate_entity_dofs_1_2": [], 
            "tabulate_entity_dofs_2_0": [0], 
            "tabulate_coordinates_0": [0.8169999918299999, 0.63033332703], 
            "num_sub_dofmaps": 0, 
        },
        "default_cell_integral": 1, 
        "cell_integral": {
            "tabulate_tensor": [0.0, 0.0, 0.0, 0.003115800031157999, 0.003115800031157999, 0.003115800031157999], 
        },
        "default_exterior_facet_integral": 0, 
        "default_interior_facet_integral": 0, 
        "default_point_integral": 0, 
    },
}
