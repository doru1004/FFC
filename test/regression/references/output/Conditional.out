
Testing form
------------
0_num_coefficients = 1
1_num_cell_domains = 1
2_num_exterior_facet_domains = 0
3_num_interior_facet_domains = 0

Testing finite_element
----------------------
4_cell_shape = 1
5_space_dimension = 6
6_value_rank = 0
7_evaluate_basis:_0 = 2.861007167858017
8_evaluate_basis:_1 = 6.188116389316917
9_evaluate_basis:_2 = 14.98411813143485
10_evaluate_basis:_3 = -20.25499781634912
11_evaluate_basis:_4 = -14.71034988724643
12_evaluate_basis:_5 = 11.93210574240425
13_evaluate_basis_all = 2.861007167858017 6.188116389316917 14.98411813143485 -20.25499781634912 -14.71034988724643 11.93210574240425
14_evaluate_basis_derivatives_0_0 = 2.861007167858017
15_evaluate_basis_derivatives_0_1 = 0.4183016632598591 -11.13728113944124
16_evaluate_basis_derivatives_0_2 = 0.02929918488607441 -0.7800958379073109 -0.7800955273458143 20.770049714109
17_evaluate_basis_derivatives_1_0 = 6.188116389316917
18_evaluate_basis_derivatives_1_1 = -16.80228443241919 3.193194233653243
19_evaluate_basis_derivatives_1_2 = 22.35954107583998 -4.24932433998028 -4.249324213791152 0.8075636705377246
20_evaluate_basis_derivatives_2_0 = 14.98411813143485
21_evaluate_basis_derivatives_2_1 = -25.05263017019036 -20.11267492536409
22_evaluate_basis_derivatives_2_2 = 20.7700505623934 16.67454763459752 16.67454763459752 13.38660866439519
23_evaluate_basis_derivatives_3_0 = -20.25499781634912
24_evaluate_basis_derivatives_3_1 = 42.10204576699874 10.33961641517088
25_evaluate_basis_derivatives_3_2 = -43.10029269019528 -13.20531909572782 -13.20531891135545 6.575877590910224
26_evaluate_basis_derivatives_4_0 = -14.71034988724643
27_evaluate_basis_derivatives_4_1 = 12.56063157739744 33.54450483314719
28_evaluate_basis_derivatives_4_2 = 1.560191565408481 -20.14377617346721 -20.14377635783958 -33.3490949197006
29_evaluate_basis_derivatives_5_0 = 11.93210574240425
30_evaluate_basis_derivatives_5_1 = -13.22606377112947 -15.82735912223852
31_evaluate_basis_derivatives_5_2 = -1.618790685187193 21.70396796518927 21.70396752843863 -8.191005649914606
32_evaluate_basis_derivatives_all_0 = 2.861007167858017 6.188116389316917 14.98411813143485 -20.25499781634912 -14.71034988724643 11.93210574240425
33_evaluate_basis_derivatives_all_1 = 0.4183016632598591 -11.13728113944124 -16.80228443241919 3.193194233653243 -25.05263017019036 -20.11267492536409 42.10204576699874 10.33961641517088 12.56063157739744 33.54450483314719 -13.22606377112947 -15.82735912223852
34_evaluate_basis_derivatives_all_2 = 0.02929918488607441 -0.7800958379073109 -0.7800955273458143 20.770049714109 22.35954107583998 -4.24932433998028 -4.249324213791152 0.8075636705377246 20.7700505623934 16.67454763459752 16.67454763459752 13.38660866439519 -43.10029269019528 -13.20531909572782 -13.20531891135545 6.575877590910224 1.560191565408481 -20.14377617346721 -20.14377635783958 -33.3490949197006 -1.618790685187193 21.70396796518927 21.70396752843863 -8.191005649914606
35_evaluate_dof_0 = 0.3079230000000001
36_evaluate_dof_1 = 0.430287
37_evaluate_dof_2 = 0.772821
38_evaluate_dof_3 = 0.59985
39_evaluate_dof_4 = 0.5310900000000001
40_evaluate_dof_5 = 0.405528
41_evaluate_dofs = 0.3079230000000001 0.430287 0.772821 0.59985 0.5310900000000001 0.405528
42_interpolate_vertex_values = 0.3079230000000001 0.430287 0.772821
43_num_sub_elements = 0

Testing finite_element
----------------------
44_cell_shape = 1
45_space_dimension = 1
46_value_rank = 0
47_evaluate_basis:_0 = 1
48_evaluate_basis_all = 1
49_evaluate_basis_derivatives_0_0 = 1
50_evaluate_basis_derivatives_0_1 = 0 0
51_evaluate_basis_derivatives_0_2 = 0 0 0 0
52_evaluate_basis_derivatives_all_0 = 1
53_evaluate_basis_derivatives_all_1 = 0 0
54_evaluate_basis_derivatives_all_2 = 0 0 0 0
55_evaluate_dof_0 = 0.5149823230336866
56_evaluate_dofs = 0.5149823230336866
57_interpolate_vertex_values = 0.5149823230336866 0.5149823230336866 0.5149823230336866
58_num_sub_elements = 0

Testing dofmap
---------------
59_needs_mesh_entities_0 = 1
60_needs_mesh_entities_1 = 1
61_needs_mesh_entities_2 = 0
62_init_mesh = 0
63_init_cell = 0
64_init_cell_finalize = 0
65_global_dimension = 20003
66_local_dimension = 6
67_max_local_dimension = 6
68_geometric_dimension = 2
69_num_facet_dofs = 3
70_num_entity_dofs_0 = 1
71_num_entity_dofs_1 = 1
72_num_entity_dofs_2 = 0
73_tabulate_dofs = 0 0 0 10001 10002 10003
74_tabulate_facet_dofs_0 = 1 2 3
75_tabulate_facet_dofs_1 = 0 2 4
76_tabulate_facet_dofs_2 = 0 1 5
77_tabulate_entity_dofs_0_0 = 0
78_tabulate_entity_dofs_0_1 = 1
79_tabulate_entity_dofs_0_2 = 2
80_tabulate_entity_dofs_1_0 = 3
81_tabulate_entity_dofs_1_1 = 4
82_tabulate_entity_dofs_1_2 = 5
83_tabulate_entity_dofs_2_0 =
84_tabulate_coordinates_0 = 0.903 0.341
85_tabulate_coordinates_1 = 0.5610000000000001 0.767
86_tabulate_coordinates_2 = 0.987 0.783
87_tabulate_coordinates_3 = 0.774 0.775
88_tabulate_coordinates_4 = 0.9450000000000001 0.5620000000000001
89_tabulate_coordinates_5 = 0.732 0.554
90_num_sub_dofmaps = 0

Testing dofmap
---------------
91_needs_mesh_entities_0 = 0
92_needs_mesh_entities_1 = 0
93_needs_mesh_entities_2 = 0
94_init_mesh = 0
95_init_cell = 0
96_init_cell_finalize = 0
97_global_dimension = 1
98_local_dimension = 1
99_max_local_dimension = 1
100_geometric_dimension = 2
101_num_facet_dofs = 0
102_num_entity_dofs_0 = 0
103_num_entity_dofs_1 = 0
104_num_entity_dofs_2 = 1
105_tabulate_dofs = 0
106_tabulate_facet_dofs_0 =
107_tabulate_facet_dofs_1 =
108_tabulate_facet_dofs_2 =
109_tabulate_entity_dofs_0_0 =
110_tabulate_entity_dofs_0_1 =
111_tabulate_entity_dofs_0_2 =
112_tabulate_entity_dofs_1_0 =
113_tabulate_entity_dofs_1_1 =
114_tabulate_entity_dofs_1_2 =
115_tabulate_entity_dofs_2_0 = 0
116_tabulate_coordinates_0 = 0.8169999918299999 0.63033332703
117_num_sub_dofmaps = 0

Testing cell_integral
---------------------
118_tabulate_tensor = 0 0 0 0.003115800031157999 0.003115800031157999 0.003115800031157999
